code_samples.md

```diff
@@ -117,13 +139,17 @@ PackageBuilder(
 Exception: The manifest has not been pushed to a remote repository. Use push_manifest() to first push the manifest to s3 or use allow_local_manifest=True if this is intentional
 ```
 
+>I was confused by this example. I would expect a package to be reproducible regardless of whether or not it lives in S3. More precisely, the package should contain enough information to verify that a dataset matches a defined state. The underlying files might no longer be available, but any consumer of the package should be able to verify if the contents match the expected hashes.
```

Comment:
The thinking was that no one else can reproduce the results if the package is local. I don't have a strong opinion about this, but I do think we need guardrails so someone doesn't think their work can be replicated when it can't